create/tasks.py

```diff
@@ -30,7 +30,7 @@
 celery.conf.broker_use_ssl = {
     'ssl_cert_reqs': ssl.CERT_NONE,
 }
-# `redis_backend_use_ssl` is an OCF patch which was proposed upstream:
+# `redis_backend_use_ssl` is an OCF patch which was merged upstream:
```

Comment:
probably don't need the comment now that it's part of upstream and documented in the regular docs?

Reply:
Yeah, I wondered about that too, I'll just remove it then.